app/test/frontend/static_files_test.dart

```diff
@@ -77,5 +77,35 @@ void main() {
         expect(f.etag.contains('mocked_hash_'), isFalse);
       });
     }
+
+    test('proper hash in css content', () {
+      final cssHashes = <String, String>{};
+      final css = cache.getFile('/static/css/style.css');
+      for (Match m in RegExp('url\\("(.*?)"\\);')
```

Comment:
Why the question mark? Why not just look for all urls...

```
RegExp('url\\([\'"](.*)[\'"]\\)')
```
Don't we want all URLs?